ppc: add vsx2 detection

gyakovlev opened the pull request:
also update gitignore while at it.

I may find couple new hwcap dumps to extend tests a bit.

gyakovlev committed:
ppc: add vsx2 detection

used by x265 and some other media libs

Signed-off-by: Georgy Yakovlev <gyakovlev@gentoo.org>

gyakovlev committed:
.gitignore: ignore hwcap-dump

Signed-off-by: Georgy Yakovlev <gyakovlev@gentoo.org>

mgorny commented:
Thanks. Do you need a release for it now, or do you want to look into the extra dumps first?

gyakovlev commented:
I've tested on all arches where it's present, and on power7 where it's not (qemu emulated).
 ok to release. 

I've already added flag to profiles and updated old news item revision.
comment about more hwcaps is just something though aloud. will do later using real HW, not emulated.
